modules/plugin/jdbc/jdbc-hana/src/main/java/org/geotools/data/hana/HanaVersion.java

```diff
@@ -23,7 +23,9 @@
  *
  * @author Stefan Uhrig, SAP SE
  */
-class HanaVersion {
+class HanaVersion implements Comparable<HanaVersion> {
```

Comment:
I'd rather introduce a new `HanaCloudVersion` class. The structure of is `<year>.<week>.<patch>-<addendum>`. The parsing of the version string would then be in that class as well.

Reply:
Done.